app/views/page/show.html.erb

```diff
@@ -10,6 +10,19 @@
 <% accordion_ctr = 0 %>
 <% page_narrow_classes = 'desktop:grid-col-8 margin-x-auto' if @page.narrow? %>
 <div id="page-builder-page" class="margin-top-5">
+  <section class="usa-section padding-y-0 margin-bottom-1 margin-top-0">
+    <div class="grid-container position-relative">
+      <div class="grid-row grid-gap">
+        <% unless @page.published? %>
+          <div class="usa-alert usa-alert--warning">
+            <div class="usa-alert__body">
+              <h3 class="usa-alert__heading">This Page will not be visible to non-admins because it is unpublished</h3>
```

Comment:
```suggestion
              <h3 class="usa-alert__heading">This page is not visible because it is not published</h3>
```
Let's simplify the language here since this page can also be visible to page group editors.